### PR TITLE
Basic Navbar

### PR DESCRIPTION
**Pull request by rushikesh3299**
Basic responsive Navbar
No functionalities & routing
https://deploy-preview-1--ayumall.netlify.app/

Animation

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,3 +1,40 @@
...
+
+  --cta-color: #fb641b;
+  --cta-color-hover: #ff8200;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+a {
```
Comment (JantuDeb) [inline src/App.css]: We will use Link from react router so no need to add any styles for `a`

**Review by JantuDeb — APPROVED** [review]
I would suggest you to use react router from start then replace  ` a href= ` with `Link` from react router.
 Another thing is that you wrote all the css for navbar. You might add css from html css e-commerce project then it will take very less time.

File: src/App.css (hunk 1 above)
Comment (rushikesh3299) [reply src/App.css]: Thank You Jantu. I removed this.

**Review by Raghav888 — APPROVED** [review]
Good you are raising small PR

**Pull request merged**